miden-lib/asm/miden/contracts/wallets/basic.masm

```diff
@@ -42,34 +46,65 @@ end
 #! - The non-fungible asset is not found in the vault.
 export.send_asset.1
     exec.account::remove_asset
-    # => [ASSET, tag, aux, note_type, RECIPIENT, ...]
+    # => [ASSET, tag, aux, note_type, RECIPIENT, PAD(5)]
 
     # Store the ASSET for later
     loc_storew.0 dropw
-    # => [tag, aux, note_type, RECIPIENT, ...]
-
-    # This procedure is written to be executed with `exec` or `call`. When this
-    # procedure is `call`ed the stack has to be carefully manipulated to avoid
-    # inserting unwanted elements between the user data. The convention is to
-    # ensure the input&output data have the same length. This code pads the
-    # stack so the output stack will be the same length as the input.
-    #
-    # The padding must be added before calling `create_note`, not after. This
-    # is because the VM stack has a minimum size of 16 elements, trying to push
-    # elements after the call to `create_note` would increase the stack in
-    # addition to the minimum 16 elements.
-    push.0 movdn.7 padw movdnw.2 padw movdnw.2
-    # => [tag, aux, note_type, RECIPIENT, 0, EMPTY_WORD, EMPTY_WORD, ...]
+    # => [tag, aux, note_type, RECIPIENT, PAD(9)]
 
     exec.tx::create_note
-    # => [note_idx, 0, EMPTY_WORD, EMPTY_WORD, ...]
+    # => [note_idx, PAD(15) ...]
 
-    padw loc_loadw.0 movup.4
-    # => [note_idx, ASSET, 0, EMPTY_WORD, EMPTY_WORD, ...]
+    movdn.4 loc_loadw.0 movup.4
+    # => [note_idx, ASSET, PAD(11) ...]
 
     exec.tx::add_asset_to_note
-    # => [note_idx, 0, EMPTY_WORD, EMPTY_WORD, ...]
+    # => [note_idx, PAD(15) ...]
+end
+
+#! Creates a note without assets.
+#!
+#! Note: since this procedure must be only called, the operand stack should be padded to the 16 
+#! elements before being called. See the corresponding issue: 
+#! https://github.com/0xPolygonMiden/miden-base/issues/685
```

Comment:
I would probably not mention the issue here as nothing should change once the issue is resolved. Maybe write this as something like:
```
Creates a new note.

The created note will not have any assets attached to it. To add assets to this note use the
`move_asset_to_note` procedure.

This procedure is expected to be invoked using a `call` instruction. It makes no guarantees
about the contents of the `PAD` elements shown below. It is the callers responsibility to make
sure these elements do not contain any meaningful data.
```